### ISSUE TITLE
Create help page: https://www.freebmd.org.uk/postems-help.html  

### ISSUE THREAD

**Issue by neslihanu** [FreeUKGen/FreeBMD2#133]

**Comment by KatyMawhood** [comment]
@PatReynolds What's the status on this?

**Comment by PatReynolds** [comment]
Not yet created @KatyMawhood

**Comment by PhilipClarke712** [comment]
This item appeared in the shared spreadsheet under the title "Add a Note (was create a Postem)" and was marked as MVP. So the existing html Help page ( https://www.freebmd.org.uk/postems-help.html) from FreeBMD was moved across to Refinery on test3 and was  modified to read "Add a Note" instead of "Postem". However, both test2 and test3 search results use "Postem", so the page as written is confused as to whether it is "Add or Note", or "Postem"; but what it should be called can be sorted out later. So the page was edited in accordance with the Marking up of Pages in Refinery guidance to have the new "look" and active verbs (rather than the passive ones used on FreeBMB).  I am also aware that a new Search Postems page has to be created, as FreeBMD uses a Perl page.

For the moment I don't propose to do any more work on this Draft Help page on Refindery: it describes what can be done on FreeBMD, but as far as I can see there is nothing operational on test2 nor test3 to describe. I'm happy to update this Draft Help page once there is something on FreeBMD2 Refinery to write about.

**Comment by PatReynolds** [comment]
There is now in-page help in test2.  It uses 'postem'.  It contains a basics of postems.

I've created a new story (#352) to cover, at a later date, the terminology change.

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds 

Can I check, do you mean test2, or 3?

I have a draft page on test3, **https://test3.freebmd.org.uk/cms/help/draft-postems** which has been sitting there since last October. I've not yet found any Postems on test3 - Barrie suggested there might, at most, be a hand-full on the data-base being used for test3 (he has a way of searching historical data-sets on FreeBMD(1) which is not something I can do). 

My draft-page  was originally written as "**Add a Note**", because that was the  name / terminology being used in the shared Google drive in May 2020. I changed it's name and all references back to **Postems** in the Autumn of 2020 because that is what test3 was using last time I looked.

Since then, other draft pages have take priority.

**Comment by PatReynolds** [comment]
Yes, test2  this is not a draft html page, it is the developer-coded page: See https://test2.freebmd.org.uk/60c1fa52bd30f23f40d07e0d/entry-information/386757/elizabeth-wilkinson-death-yorkshire-north-riding-northallerton-v24-p239?locale=en for an example.

**Comment by PatReynolds** [comment]
It is in Test3, too, but you have to know to visit 'more options' to add one.

https://test3.freebmd.org.uk/60c23207bd30f25def94f29f/entry-information/4086897/george-alcock-death-yorkshire-north-riding-stockton-c-v24-p173?locale=en has a postem that I've just added - you should see that when you visit the page.

**Comment by AlOneill** [comment]
@PatReynolds 

The opening blurb has a couple of typos:
1. I think that "For example, you you know " should be "For example, do you know "
2. "Below are the list of postems" should be either "Below are the postems" or "Below is the list of postems" or, less awkwardly, "The list of postems is below:" or "These are the postems:" or similar.

**Comment by AlOneill** [comment]
@PatReynolds 
Also, there needs to be some space between the navigation markers and the `grid__item` that holds the postems: suggest changing the class of `push--bottom` to `push--ends`

**Comment by PatReynolds** [comment]
@Vino-S will need to fix, as this is not arefinery page.

**Comment by PhilipClarke712** [comment]
Hi @PatReynolds 
Thanks,
Two comments: (1) that hyperlink does not work - I got a "We're sorry, but something went wrong with processing your request. A report has been produced and sent to the webmaster. Please return to your prior activity but try to avoid doing the exact same action again".  **But** I did the search using the parameters in the hyperlink and found George's postem.

(2) Postems are **hidden** - you don't see them unless you use that dropdown menu that you told me about. On BMD(1) I was looking for some records and there you get a symbol - saying there is something to look at. George is not my relative so someone saying - "he was my grandfather's best man" might not interest me, but they might be saying the GRO has miss-indexed this as  **Stockton** and it should be **Stockport** (not a very likely example) and that could be of great interest to me. I'd like an indication that there is something to look at.

**Comment by PatReynolds** [comment]
Stockport / Stockton should be a correction rather than a postem ... But I get your point.  

I don't, personally, think the current (test3) version is superior to the earlier (test2) version, but we shall see what users think!

**Comment by PhilipClarke712** [comment]
I was careful to blame the GRO for the error - "my" records were correctly transcribed by FreeBMD, so correction is not an option - but I might ask for a system entry (Barrie can do that on BMD1 - not sure about BMD2).

I've been using FreeBMD for almost 20 years, Ancestry & Findmypast for perhaps half that time and FamilySearch for a bit longer - my default BMD search is on BMD(1) because I can do things I can't on others. Yes "my comfort zone has been disturbed" - I need time to adjust, after than using BMD1 may feel "strange" - but I will still need be using for it my transcriptions.

**Comment by DeniseColbert** [comment]
- [ ] Fix 'you you' in Postem text
- [ ] Make postem entry field display by default
- [ ] Change button text to 'Submit Postem'

**Comment by Vino-S** [comment]
deployed to production
